metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -1012,6 +1201,33 @@ def gen_database_key(self, database: str) -> DatabaseKey:
             backcompat_env_as_instance=True,
         )
 
+    def gen_platform_resource(
+        self, catalog: str, tag: LakeFormationTag
+    ) -> Iterable[MetadataWorkUnit]:
+        if self.ctx.graph and self.platform_resource_repository:
+            platform_resource_id = LakeFormationTagPlatformResourceId.from_tag(
+                platform_instance=self.source_config.platform_instance,
+                platform_resource_repository=self.platform_resource_repository,
+                catalog=catalog,
+                tag=tag,
+            )
+            logger.info(f"Created platform resource {platform_resource_id}")
+
+            lf_tag = LakeFormationTagPlatformResource.get_from_datahub(
+                platform_resource_id, self.platform_resource_repository, False
+            )
+            if (
+                tag.to_datahub_tag_urn().urn()
+                not in lf_tag.datahub_linked_resources().urns
+            ):
+                lf_tag.datahub_linked_resources().add(tag.to_datahub_tag_urn().urn())
+                platform_resource = lf_tag.as_platform_resource()
+                for mcp in platform_resource.to_mcps():
+                    yield MetadataWorkUnit(
```

Comment:
Can we also mint the TagKey? To make sure it gets created as an entity on DH.

Reply:
Ingestion framework generates automatically TagKeys